"Cannot find module 'eslint-config-xo-react'"

If I try to use `eslint-config-xo-react` the linter complains it can't load the module.

The stack trace goes through `~/.atom`, I assume it's not trying the node_modules of the project. From the command line, xo works fine.

Am I missing some configuration?

package.json:
```
  "devDependencies": {
    "eslint": "^1.9.0",
    "eslint-config-xo-react": "^0.3.0",
    "eslint-plugin-react": "^3.9.0",
    "xo": "^0.11.1"
  },
  "xo": {
    "esnext": true,
    "semicolon": false,
    "env": [
      "browser",
      "mocha",
      "webpack"
    ],
    "extends": "eslint-config-xo-react"
  },
```

No change, I also updated `xo` to 0.11.2 in the project and restarted Atom.

I extracted a stacktrace via the devtools:

```
Error: Cannot find module 'eslint-plugin-react
  at Module._resolveFilename (module.js:336:15
  at Function.Module._resolveFilename (/Users/wmertens/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52
  at Function.Module._load (module.js:286:25
  at Module.require (module.js:365:17
  at require (module.js:384:17
  at /Users/wmertens/.atom/packages/linter-xo/node_modules/xo/node_modules/eslint/lib/cli-engine.js:124:
  at Array.forEach (native
  at loadPlugins (/Users/wmertens/.atom/packages/linter-xo/node_modules/xo/node_modules/eslint/lib/cli-engine.js:115:21
  at processText (/Users/wmertens/.atom/packages/linter-xo/node_modules/xo/node_modules/eslint/lib/cli-engine.js:202:5
  at CLIEngine.executeOnText (/Users/wmertens/.atom/packages/linter-xo/node_modules/xo/node_modules/eslint/lib/cli-engine.js:678:26
  at exports.lintText (/Users/wmertens/.atom/packages/linter-xo/node_modules/xo/index.js:107:16
  at /Users/wmertens/.atom/packages/linter-xo/index.js:17:
  at exports.allowUnsafeNewFunction (/Users/wmertens/.atom/packages/linter-xo/node_modules/loophole/lib/loophole.js:25:14
  at Object.lint (/Users/wmertens/.atom/packages/linter-xo/index.js:16:2
  at promises.push.Promise.then._this.emitter.emit.linter (/Users/wmertens/.atom/packages/linter/lib/linter-registry.js:63:26
  at /Users/wmertens/.atom/packages/linter/lib/linter-registry.js:62:
  at Set.forEach (native
  at LinterRegistry.lint (/Users/wmertens/.atom/packages/linter/lib/linter-registry.js:60:18
  at /Users/wmertens/.atom/packages/linter/lib/linter.coffee:98:
  at Emitter.module.exports.Emitter.emit (/Users/wmertens/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11
  at /Users/wmertens/.atom/packages/linter/lib/editor-linter.js:80:
                                                                                       at /Users/wmertens/.atom/packages/linter/lib/helpers.coffee:23:18
```

--- a/index.js
+++ b/index.js
@@ -3,6 +3,7 @@ import path from 'path';
 import {CompositeDisposable} from 'atom';
 import {allowUnsafeNewFunction} from 'loophole';
 import setText from 'atom-set-text';
+import pkgDir from 'pkg-dir';
 
 let lintText;
 allowUnsafeNewFunction(() => {
@@ -13,12 +14,14 @@ function lint(textEditor) {
 	const filePath = textEditor.getPath();
 	let report;
 
+	const dir = pkgDir.sync(path.dirname(filePath));
+
 	// ugly hack to workaround ESLint's lack of a `cwd` option
 	const defaultCwd = process.cwd();
-	process.chdir(path.dirname(filePath));
+	process.chdir(dir);
 
 	allowUnsafeNewFunction(() => {
-		report = lintText(textEditor.getText(), {cwd: path.dirname(filePath)});
+		report = lintText(textEditor.getText(), {cwd: dir});
 	});
 
 	process.chdir(defaultCwd);

--- a/package.json
+++ b/package.json
@@ -34,6 +34,7 @@
     "atom-package-deps": "^3.0.0",
     "atom-set-text": "^1.0.1",
     "loophole": "^1.1.0",
+    "pkg-dir": "^1.0.0",
     "xo": "^0.11.1"
   },
   "package-deps": [

